xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecCCText.cpp

```diff
@@ -89,6 +89,8 @@ OverlayMessage CDVDOverlayCodecCCText::Decode(DemuxPacket* pPacket)
   {
     TagConv.ConvertLine(text);
     TagConv.CloseTag(text);
+    // We have to remove them all because it causes to display empty box "tofu"
+    StringUtils::Replace(text, "\r", "");
```

Comment:
wondering if it makes sense to move this to the tag converter either as part of the `CloseTag` or in some new method there

Reply:
this should be removed when will be completed the rework of the CC decoders, i can add a comment as reminder
also because the `CDVDSubtitleTagSami TagConv;` here will no longer required, probably even now is not properly used

Reply:
Yeah please add doxygen TODOs if possible

Reply:
done